coreprofile/10/jakarta-coreprofile-spec-10.0.html

```diff
@@ -0,0 +1,793 @@
+<div class="paragraph">
+<p>The Core Profile is targeted at developers of
+modern cloud applications with focus on a minimal set of APIs
+that support microservices with a focus on enabling implementations
+that make use of ahead of time complitation build to minimize startup
+time and memory footprint.</p>
+</div>
+<div class="paragraph">
+<p>It is worth noting that Core Profile products are allowed to ship with more technologies than the
+required ones. It is conceivable that products will offer a choice at
+installation time/or build time between different configurations, some richer in
+extensions, or even allow for complete customization beyond the required
+core (“à la carte” installation).</p>
+</div>
+</div>
+<div class="sect2">
+<h3 id="determining-applicable-requirements"><a class="anchor" href="#determining-applicable-requirements"></a>1.2. Determining Applicable Requirements</h3>
+<div class="admonitionblock note">
+<table>
+<tr>
+<td class="icon">
+<i class="fa icon-note" title="Note"></i>
+</td>
+<td class="content">
+Profile definitions can be quite terse, amounting to little more than a list of required technologies and a
+(possibly empty) set of additional requirements, beyond those entailed by all the referenced specifications.
+</td>
+</tr>
+</table>
+</div>
+<div class="paragraph">
+<p>As dictated by the general rules for Jakarta EE profiles in the Platform specification, products that implement the Core
+Profile must honor:</p>
+</div>
+<div class="olist arabic">
+<ol class="arabic">
+<li>
+<p>all requirements of the common requirements section that applies to all profiles;</p>
+</li>
+<li>
+<p>all requirements of this specification;</p>
+</li>
+<li>
+<p>all requirements of the individual component specifications;</p>
+</li>
+<li>
+<p>all requirements in the Jakarta EE Platform specification that are conditional on the presence of a specific
+technology or combinations of technologies.</p>
+</li>
+</ol>
+</div>
+<div class="paragraph">
+<p>Let’s look at some examples of requirements from each grouping.</p>
+</div>
+<div class="paragraph">
+<p>For the first one, the Jakarta EE Platform
+specification mandates support for Java&#8482; Platform, Standard Edition 11 API.  Because of this, all Core Profile 10.0 APIs must support the Java&#8482; Platform, Standard Edition 11 API.</p>
+</div>
+<div class="paragraph">
+<p>In the second category one can point out the
+that Jakarta XML Binding support is not required for Jakarta RESTful Web Services in the Core Profile.
+(<a href="#additional_requirements">see Additional Requirements</a>).</p>
+</div>
+<div class="paragraph">
+<p>The third category is hopefully
+self-explanatory. For example, Core Profile products must implement the Jakarta RESTful Web Services API, which in turn means they need to satisfy all the requirements listed in the Jakarta RESTful Web Services 3.1 specification.</p>
```

Comment:
is the presence of the Rest spec version number here required? Isn't it enough to have it listed in the _Required Components_ section?

Reply:
Yes, dropped the version